Module/utc/selanflembek.utc.json

```diff
@@ -723,5 +723,81 @@
   "Wis": {
     "type": "byte",
     "value": 10
+  },
+  "xAppearance_Head": {
```

Comment:
I haven't checked all of these files but it looks like something added all these properties to all creatures. Please review & remove.